### ISSUE TITLE
Metamod

### ISSUE THREAD

**Issue by ramonberrutti** [gabrielfgularte/sourcemod-compiler#1]
Hi bro, you don't neeed to download metamod, also I recommend to download from the last master of sourcemod and not to keep with a version.

**Comment by gabrielfgularte** [comment]
Hi @ramonberrutti ! Thanks!

1 - Sourcemod doesn't require metamod to work?
2 - Do you know if there's a link for the latest sourcemod version?

**Comment by ramonberrutti** [comment]
To work yes, but you are doing a docker for compile, or you just want to all that files go to a server?
To get the last you can git clone the project or download master.zip

**Comment by gabrielfgularte** [comment]
Good. I'll leave metamod outside.

Seems that there's no way to compile plugins by cloning the project or downloading the master.zip because they're both source code. I think that I need the release download for that, and I don't know how to get the latest release.

Another option would be download the source code and build it. But I don't know C or C++... so I guess that I'm out of bullets here. Do you know C or have any ideas?

**Comment by ramonberrutti** [comment]
Yes, let me finish something of the office here, and I can help you
